File: Frameworks/Sakura/cpp/plugins/vnragent/growl.cc
// growl.h
// 2/1/2013 jichi

#include "growl.h"
#include "driver/rpcclient.h"

void growl::msg(const QString &msg) { if (auto p = RpcClient::instance()) p->growlMessage(msg); }
void growl::warn(const QString &msg) { if (auto p = RpcClient::instance()) p->growlWarning(msg); }
void growl::error(const QString &msg) { if (auto p = RpcClient::instance()) p->growlError(msg); }
void growl::notify(const QString &msg) { if (auto p = RpcClient::instance()) p->growlNotification(msg); }

// EOF

File: Frameworks/Sakura/py/apps/browser/growl.py
# coding: utf8
# growl.py
# 10/28/2012 jichi

from sakurakit.skdebug import dprint, dwarn, derror

def show(async=False): pass
def msg(text, async=False): dprint(text)
def warn(text, async=False): dwarn(text)
def error(text, async=False): derror(text)
def notify(text, async=False): dprint(text)

# EOF
